### ISSUE TITLE
Accessibility issue when using p-dropdown inside accordian

### ISSUE THREAD

**Issue by priyam-shah-hiwi** [primefaces/primeng#14905]
### Describe the bug

https://github.com/primefaces/primeng/assets/146003652/4bd28db7-f80e-43e6-a93c-137697d7296e

When using dropdown inside accordian tab, the keyboard accessibility like tab, arrow-down arrow-up does not work. The arrow-up event changes the focus from the dropdown to the accordian. As seen from the video when i move the focus to payment type and the arrow keys move the focus to the parent element ie arrodian.

### Environment

Angular 17.0.7
PrimeNg 17.8.0

### Angular version

17.0.7

### PrimeNG version

17.8.0

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

v18.16.1

### Browser(s)

Chrome, Edge

### Steps to reproduce the behavior

1. Create dropdown inside accordian tab
2. press tab to move on the dropdown
3. press enter / space to open the options
4. press arrow-up / arrow-down 
focus changes to accodian

### Expected behavior

Focus should stay on the dropdown and the options should be accessed using key-up and key-down
(issue got introduced in primeng 17, does not occur in ngprime 13)
